### PR TITLE
Task dev 7

### PR DESCRIPTION
**Pull request by ineskavalenka**
Here are dev-6 and dev-7 working together :)

### CODE REVIEW

File: SortingCenter.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
```
Comment (AlexanderSoroka) [inline SortingCenter.java:1]: Sort in triangle task?

File: TriangleAnalyzer.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    sideB = new BigDecimal(sides[1]);
+    sideC = new BigDecimal(sides[2]);  
+  } 
+  
+  /**
+   * Checks, if the triangle is equilateral.
+   * 
+   * @return true if yes, false if no.
+   */
+  public boolean checkIfEquilateral() {
```
Comment (AlexanderSoroka) [inline TriangleAnalyzer.java:25]: there should be separate classes for each triangle type

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: SortingCenter.java (hunk 1 above)
Comment (ineskavalenka) [reply SortingCenter.java:1]: Yep. Triangle sides are sorted by nondecreasing for shorter and easier analysis. And I'm going to use it in further tasks.

File: ConsoleReader.java [hunk 3]
```diff
@@ -17,17 +17,33 @@
...
   }
+
+  /**
+   * Requests the sides of a triangle from the console,
+   * but doesn't check if they are entered correctly.
+   *
+   * @return the entered sides of a triangle.
+   */    
+    public String[] readTriangle() { 
+    Scanner input = new Scanner(System.in);
```
Comment (AlexanderSoroka) [inline ConsoleReader.java:28]: codestyle )

File: Triangle.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  
+  /**
+   * Triangle constructor; makes a triangle from the sequence of its sides.
+   * Sides of the created triangle are sorted by nondecreasing, even if the input sequence is not.
+   *
+   * @param sidesSequence a sequence of triangle sides
+   * @throws IllegalArgumentException if the sequence has not three sides.
+   * @throws NumberFormatException if sides are not numbers.
+   */
+  public Triangle(String[] sidesSequence) {
```
Comment (AlexanderSoroka) [inline Triangle.java:16]: Argument must be number array not string array - in that case Triangle will not verify number format like it does it current implementation.

Just think - triangle verifies number format, sounds weird, right?

File: Triangle.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    SequenceAnalyzer analyzer = new SequenceAnalyzer();
+    if (!analyzer.checkIfTriangleSides(sidesSequence)) {
```
Comment (AlexanderSoroka) [inline Triangle.java:18]: nope, SequenceAnalyzer must know nothing about triangle, only triangle by itself can know when it exists

File: Triangle.java [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      throw new IllegalArgumentException();
+    }
+    for (int i = 0; i < NUMBER_OF_SIDES; i++) {
+      sides[i] = Double.parseDouble(sidesSequence[i]); 
+    } 
+    SortingCenter sorter = new SortingCenter();
```
Comment (AlexanderSoroka) [inline Triangle.java:24]: I'm not sure if you really need to sort it

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

**Pull request closed without merging**